install_requirements.py

```diff
@@ -113,11 +119,24 @@ def python_is_compatible():
 
 # pip packages needed by exir.
```

Comment:
Can you keep the comment for why the version is not needed when fetching nightly here?

Reply:
Oh I thought that I have mention it in the argument section above.  But I could add the comment back if you think it's closed to the code